src/main/java/com/dev/cinema/controllers/HelloController.java

```diff
@@ -0,0 +1,15 @@
+package com.dev.cinema.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@ResponseBody
```

Comment:
do we need this annotation in pair with @RestController?

Reply:
https://javarevisited.blogspot.com/2017/08/difference-between-restcontroller-and-controller-annotations-spring-mvc-rest.html